bugwarrior/command.py

```diff
@@ -15,6 +16,7 @@
 )
 
 import logging
+logging.basicConfig()
```

Comment:
Hmm, well we can't quite do that because it depends on the logging settings in the configuration!